packages/discovery-provider/ddl/functions/handle_comment.sql

```diff
@@ -26,7 +26,7 @@ begin
   end if;
 
   begin
-    if new.is_delete is false then
+    if new.user_id != owner_user_id then
```

Comment:
in this case new.is_delete isn't necessary because there's `on conflict do nothing` but other cases it's actually important to avoid things like double counting